### DISCUSSION TITLE
Configuring nortfications to use aws sns/sqs

### DISCUSSION THREAD

**Discussion by whiteninja76** [DependencyTrack/dependency-track#4434] [Q&A]
Has anyone done this before? just looking for a little guidance on configuring dependency-track to utilise aws sqs/sns for sending notifications on new vulnerabilities when they are found .

**Comment by nscuro** [comment]
You could use Bento to forward Webhook notifications from DT to the destination of your choosing: https://warpstreamlabs.github.io/bento/

It's just a small binary and easy to operate.

**Comment by whiteninja76** [comment]
Was hoping there was something out of the box(with out using 3rd party repos/code)
